### ISSUE TITLE
[theming] unable to successfully display custom file icons in electron

### ISSUE THREAD

**Issue by vince-fugnitto** [eclipse-theia/theia#6834]
**Description**

As part of #6475, it is now possible to change the application's file icon theme.
However, when running in Electron the icons are not properly displayed:

_Default Icon (Theia)_:

<div align='center'>

![image](https://user-images.githubusercontent.com/40359487/71829330-60672980-3072-11ea-8c10-28e8c3a045d1.png)

</div>

_Seti Icon (Theia)_:

<div align='center'>

![image](https://user-images.githubusercontent.com/40359487/71829386-82f94280-3072-11ea-88b7-67e0fad8615a.png)

</div>

For reference, the browser version works correctly:

<div align='center'>

![image](https://user-images.githubusercontent.com/40359487/71829445-a9b77900-3072-11ea-8e67-019bc2ad1c4f.png)

</div>

**Steps to Reproduce**:

1. install https://registry.npmjs.org/@theia/vscode-builtin-vscode-theme-seti/-/vscode-builtin-vscode-theme-seti-0.3.0-next.2111655b67.tgz  (`Seti` Icon Theme) as a plugin
2. start the electron application
3. change file icon theme to `Seti` using the command `Preferences: File Icon Theme`

**Comment by owlJaeger** [comment]
My testing resulted in finding out that the `seti.woff` font-file will not be loaded as a `@font-face` and thus the icons won't be shown. I've already found a workaround but I am looking for a permanent solution.

If someone really needs these icons to work in his electron app, one possible workaround is to add this to the `<head></head>` section of the `lib/index.html` file:
```
@font-face {
  font-family: 'seti';
  src: url("file:///<path to your project>/plugins/vscode-builtin-icon-theme-seti/extension/icons/seti.woff") format('woff');
  font-weight: normal;
  font-style: normal;
      }
```

**Comment by vince-fugnitto** [comment]
Closed by #7583
